server/game/cards/13.1-AtG/OldGate.js

```diff
@@ -0,0 +1,30 @@
+const DrawCard = require('../../drawcard.js');
+const TextHelper = require('../../TextHelper');
+
+class OldGate extends DrawCard {
+    setupCardAbilities(ability) {
+        this.plotModifiers({
+            gold: 1
+        });
+        this.action({
+            title: 'Sacrifice to draw 2 cards',
+            condition: () => this.allCharactersHaveStarkAffiliation(),
+            phase: 'challenge',
+            cost: ability.costs.sacrificeSelf(),
+            handler: () => {
+                let cards = this.controller.drawCardsToHand(2).length;
+                this.game.addMessage('{0} sacrifices {1} to draw {2} cards', 
```

Comment:
Since you're using the TextHelper already, you don't need the additional 'cards' in the message format. It will end up outputting something like `'Player 1 sacrifices Old Gate to draw 2 cards cards'`